### PR TITLE
Add linting

### PR DESCRIPTION
**Pull request by CodeSandwich**

### CODE REVIEW

File: src/test/ERC20Pool.t.sol [hunk 1]
```diff
@@ -7,16 +7,15 @@ import {ERC20Pool, Pool} from "../ERC20Pool.sol";
 import {IERC20, ERC20PresetFixedSupply} from "openzeppelin-contracts/token/ERC20/presets/ERC20PresetFixedSupply.sol";
 
 contract ERC20PoolTest is EthPoolTest {
-
     ERC20Pool private pool;
 
     function setUp() public override {
-        IERC20 erc20 = new ERC20PresetFixedSupply("test", "test", 10 ** 6 * 1 ether, address(this));
+        IERC20 erc20 = new ERC20PresetFixedSupply("test", "test", 10**6 * 1 ether, address(this));
```
Comment (manuelpolzhofer) [inline src/test/ERC20Pool.t.sol:13]: interesting. Pow operation without space and multiplication with space

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 install_solc:; nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions.solc_0_8_7
-update      :; dapp update
+dapp_update :; dapp update
+yarn_install:; yarn install
 
 build       :; dapp build
 clean       :; dapp clean
 
+prettier    :; yarn run prettier
+lint        :; yarn run lint
```
Comment (manuelpolzhofer) [inline Makefile:10]: 👍

File: README.md [hunk 3]
```diff
@@ -5,7 +5,17 @@
 Radicle Streaming uses [dapp.tools](https://github.com/dapphub/dapptools) for development. Please install the `dapp` client. Then, run the following command to install the dependencies:
 
 ```bash
-dapp update
+make install
+```
+
+### Run linter
+```bash
+yarn run lint
```
Comment (manuelpolzhofer) [inline README.md:13]: maybe all commands with the makefile?

```suggestion
make lint
```

File: README.md [hunk 4]
```diff
@@ -5,7 +5,17 @@
...
+```
+
+### Run prettier
+```bash
+yarn run prettier
```
Comment (manuelpolzhofer) [inline README.md:18]:
```suggestion
make prettier
```

**Review by manuelpolzhofer — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (CodeSandwich) [reply README.md]: Fixed

**Pull request merged**